btr-api/src/btr_api/services/entity.py

```diff
@@ -55,14 +55,15 @@ def init_app(self, app: Flask):
         self.svc_url = app.config.get('LEGAL_SVC_URL')
         self.timeout = app.config.get('LEGAL_SVC_TIMEOUT', 20)
 
-    def get_entity(self, user_jwt: JwtManager, identifier: str) -> requests.Response:
-        """get the entity info for the given business identifier."""
+    def get_entity_info(self, user_jwt: JwtManager, path: str) -> requests.Response:
+        """Get the entity info for the given path."""
         try:
             # make api call
             token = user_jwt.get_token_auth_header()
 
             headers = {'Authorization': 'Bearer ' + token, 'Content-Type': 'application/json'}
-            resp = requests.get(url=self.svc_url + '/businesses/' + identifier,
+            self.app.logger.debug('Getting legal-api data for: %s', path)
+            resp = requests.get(url=self.svc_url + '/businesses/' + path,
```

Comment:
What do the path can consist off ?
I understand this path is suffix after that describes business and its sub information, like address or ?

Reply:
yeah I will add a comment so its clear. It will be identifier + / + sub info suffix (i.e. address)

Reply:
As long as its understood that its a url suffix I think we are good, whatever is easiest.